lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
So, eslint only started supporting glob-based rule overrides (i.e. disable a specific rule in all files that match a glob) very recently (v4.x).  `eslint-airbnb-config` hasn't been updated to work with eslint v4.  There's [an issue on airbnb/javascript to add eslint v4 support](https://github.com/airbnb/javascript/issues/1447) and it seems people are working on it.  But it's not done yet.

I think it would be great to add the override to `.eslintrc.js` once `eslint-airbnb-config` supports eslint v4.  But in the mean time these file-wide disables will work.